Makefile

```
@@ -41,6 +41,10 @@ setup-venv:
 diff-format:
```

```suggestion
diff-format:  ## Format Python code changes
```

To add this to the help print by `make` with no targe.